hibernate-core/src/test/java/org/hibernate/orm/test/sql/exec/HqlUpdateExecutionTests.java

```diff
@@ -85,9 +85,9 @@ public void testSimpleUpdateWithData(SessionFactoryScope scope) {
 	@Test
 	public void testSimpleRestrictedUpdate(SessionFactoryScope scope) {
 		scope.inTransaction(
-				session -> session.createQuery( "update BasicEntity set data = :p where data = :filter" )
+				session -> session.createQuery( "update BasicEntity set data = :p where data = :thefilter" )
```

Comment:
You have to add this token to the `identifier` rule at the end of `HqlParser.g4` to avoid the changes in this file

Reply:
Thanks, did not know this.